02-s3/README.md

```diff
@@ -224,10 +241,14 @@ file and read "private.txt".
 _What do you see when you try to read the existing bucket policy before you
 replace it?_
 
+> A: I didn't see one to replace
+
```

Comment:
This is introduced with `s3api` in a later section here.

`aws s3api put-object-acl --acl bucket-owner-full-control s3://my-bucket/private.txt`